### ISSUE TITLE
Latest changes to elens module breaks C/R

### ISSUE THREAD

**Issue by vkbo** [SixTrack/SixTrack#1064]
Some error logging to the lerr output and crlog under the CR flag in the elens module is breaking CR build.

```
/home/vkbo/Code/SixTrack/Fork/build/SixTrack_cmakesix_CR_PYTHIA_gfortran_debug/elens.f90:2012:13:

 2012 |   write(crlog,"(a,i0,a)") "CR_POINT> ERROR Writing C/R file fort.",fileUnit," in ELENS"
      |             1
Error: Symbol ‘crlog’ at (1) has no IMPLICIT type
/home/vkbo/Code/SixTrack/Fork/build/SixTrack_cmakesix_CR_PYTHIA_gfortran_debug/elens.f90:2011:12:

 2011 |   write(lerr, "(a,i0,a)") "CR_POINT> ERROR Writing C/R file fort.",fileUnit," in ELENS"
      |            1
Error: Symbol ‘lerr’ at (1) has no IMPLICIT type
/home/vkbo/Code/SixTrack/Fork/build/SixTrack_cmakesix_CR_PYTHIA_gfortran_debug/elens.f90:1990:13:

 1990 |   write(crlog,"(a,i0,a)") "CR_CHECK> ERROR Reading C/R file fort.",fileUnit," in ELENS"
      |             1
Error: Symbol ‘crlog’ at (1) has no IMPLICIT type
/home/vkbo/Code/SixTrack/Fork/build/SixTrack_cmakesix_CR_PYTHIA_gfortran_debug/elens.f90:1989:12:

 1989 |   write(lerr, "(a,i0,a)") "CR_CHECK> ERROR Reading C/R file fort.",fileUnit," in ELENS"
      |            1
Error: Symbol ‘lerr’ at (1) has no IMPLICIT type
```

@amereghe

**Comment by vkbo** [comment]
I needed to push a fix for SCATTER block parsing, so fixing this, and one of the elens tests as well.

**Comment by amereghe** [comment]
Closing issue as it has been solved with PR #1065
